pyathena/connection.py

```diff
@@ -136,8 +138,15 @@ def __init__(
                 profile_name=self.profile_name,
                 **self._session_kwargs,
             )
+
+        self.config: Optional[config.Config] = config
```

Comment:
```suggestion
        self.config: Optional[config.Config] = config if config else config.Config()
```
It may be safer to take None as an argument and initialize it in the None case to handle config more safely. (In consideration of the case where None is specified in the argument config.)

Reply:
I added the changes, but I also simplified the import
`from botocore.config import Config`